modules/nginx/src/test/java/org/testcontainers/junit/SimpleNginxTest.java

```diff
@@ -20,18 +17,14 @@
  */
 public class SimpleNginxTest {
 
-    @Rule
-    public NginxContainer nginx = new NginxContainer()
-            .withCustomContent(System.getProperty("user.home") + "/.tmp-test-container");
+    private static File contentFolder = new File(System.getProperty("user.home") + "/.tmp-test-container");
```

Comment:
Why use home directory - when test runs in developer computer it will mess it up. How about temp directory using standard jvm way?